internal/controller/sleepinfo/sleepinfo_controller.go

```diff
@@ -74,7 +74,7 @@ func (r *SleepInfoReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 
 	sleepInfo, err := r.getSleepInfo(ctx, req)
 	if err != nil {
-		log.Error(err, "unable to fetch sleepInfo")
+		log.V(8).Info(fmt.Sprintf("%s: unable to fetch sleepInfo", err.Error()))
```

Comment:
It’s better to keep the log message constant (for searchability) and include the error as a key-value pair.

With this logger, you can do something like:

```
log.V(8).Info("unable to fetch sleepInfo", "err", err.Error()))
```